apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/test/java/gov/cms/bfd/pipeline/ccw/rif/load/RifLoaderIT.java

```diff
@@ -245,17 +244,17 @@ public void failOnUpdateBeneficiaryBeforeInsert() {
     Stream<RifFile> editedSample = editSamples(samplesStream, fileEditor);
 
     // Load the edited sample to verify that it fails, as expected.
-    AssertionFailedError thrown =
+    IllegalStateException thrown =
         assertThrows(
-            AssertionFailedError.class,
+            IllegalStateException.class,
             () -> {
               loadSample(
                   "SAMPLE_A, bene only, UPDATE",
                   CcwRifLoadTestUtils.getLoadOptions(),
                   editedSample);
             });
 
-    assertTrue(thrown.getMessage().contains("Load errors encountered"));
+    assertTrue(thrown.getMessage().contains("Fatal error during rif file processing"));
```

Comment:
This changed because we're not throwing the exact exception that the subthread hit anymore, we're throwing the custom one when we have a fatalError reported in any subthread. (The original exception in the subthread wasn't particularly valuable, and it's logged anyway)